asynq/tools.py

```diff
@@ -282,6 +283,12 @@ def dirty():
 
     return decorator
 
+async def _asyncio_sleep(secs: float) -> None:
+    await asyncio.sleep(secs)
+
+@asynq(asyncio_fn=_asyncio_sleep)
```

Comment:
```suggestion
@asynq(asyncio_fn=asyncio.sleep)
```
I don't think you need the wrapper function?